### PR TITLE
Start modularizing the styles

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
**Reviewer:** @laurengarcia 
**CC:** @jdorweiler 

## Description:

Start modularizing the popup styles: separate files for different popup elements (vars, buttons, etc) which are then included into a main.scss file.

That file is compiled by the sass Grunt task, into public/css/popup.css

## Steps to test this PR:
Coming soon
1. Run `grunt`
2. Load extension in Chrome
3. Open the popup
4. Make sure the styles look exactly the same as before; navigate to some webpages, hover - everything should look exactly the same

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: css/lists.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
+ol, ul {
+  list-style: none;
+}
+
+ul {
+  padding: 0 !important;
+  margin: 0 !important;
+}
```
Comment (laurengarcia) [inline css/lists.scss]: These should go into a normalize file as they are more normalize/reset rules.
https://necolas.github.io/normalize.css/

File: css/popup.scss [hunk 2]
```diff
@@ -19,31 +19,12 @@ time, mark, audio, video {
...
-q:before, q:after {
-  content: '';
-  content: none;
-}
-table {
-  border-collapse: collapse;
-  border-spacing: 0;
-}
-
-
```
Comment (laurengarcia) [inline css/popup.scss]: These should go into a normalize file as they are more normalize/reset rules.
https://necolas.github.io/normalize.css/

File: css/popup.scss [hunk 3]
```diff
@@ -54,311 +35,11 @@ html {
 }
 
 body {
+  line-height: 1;
   font-family: "Helvetica Neue", sans-serif !important;
   font-weight: normal;
   border: 0;
   margin: 0;
   padding: 0;
 }
```
Comment (laurengarcia) [inline css/popup.scss]: Everything except for font-family here needs to go in a normalize file. The normalize file will have its own line-height rule: 
https://github.com/necolas/normalize.css/blob/master/normalize.css#L12

File: css/search_form.scss [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  height: 27px;
+  width: 21px;
+
+  background-image: url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IiB2aWV3Qm94PSIwIDAgMjAgMjAiIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDIwIDIwOyIgeG1sOnNwYWNlPSJwcmVzZXJ2ZSI+PGcgaWQ9ImNsZWFyIj48Zz48Zz48cG9seWdvbiBzdHlsZT0iZmlsbC1ydWxlOmV2ZW5vZGQ7Y2xpcC1ydWxlOmV2ZW5vZGQ7ZmlsbDojODg4ODg4OyIgcG9pbnRzPSIxNi41LDQuNyAxNS44LDQgMTAuNSw5LjMgNS4yLDQgNC41LDQuNyA5LjgsMTAgNC41LDE1LjMgNS4yLDE2IDEwLjUsMTAuNyAxNS44LDE2IDE2LjUsMTUuMyAxMS4yLDEwIi8+PC9nPjwvZz48L2c+PC9zdmc+) !important;
+  background-repeat: no-repeat;
+
+
+  display: none;
+  cursor: pointer;
+}
```
Comment (laurengarcia) [inline css/search_form.scss]: Lots of empty lines in this file inside of rule sets. Pls delete them.

**Comment by laurengarcia** [comment]
Generally, modularized files are not supposed to be specific to any one component (such as a selector id). Many of these rules are prefixed with `#advanced` for example. This makes whatever rules that follow #advanced not modular. So a rule like `#advanced button` could become something more general like `button.foo`. This way you can add .foo class to buttons inside of the #advanced container, for example, without limiting that class' use to inside of #advanced. Ping me tomorrow if you have more questions.

**Comment by russellholt** [comment]
closing, done elsewhere.

**Pull request closed without merging**